### PR TITLE
Add internationalisation with i18next for React

### PR DESCRIPTION
**Pull request by RubenSibon**

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    return {
+      ...acc,
+      [cur]: {
+        translations,
+      },
+    }
+  }, {}),
+  ns: ['translations'],
+  defaultNS: 'translations',
+  // debug: true,
```
Comment (TerryvanWalen) [inline src/i18n/index.js:32]: Remove?

File: test/utils/withApp.tsx [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+import i18n from '../../src/i18n'
 import { ROUTES } from '../../src/routes'
 
 export const withApp = (path: string) => {
   const router = createMemoryRouter(ROUTES, {
     initialEntries: [path],
   })
 
-  return withQueryClient(withAppContext(<RouterProvider router={router} />))
+  i18n.changeLanguage('nl')
```
Comment (TerryvanWalen) [inline test/utils/withApp.tsx:15]: Should we add a test whether each language or at least one language is available?

File: src/pages/LoadUnload/components/MapSettingsDisplay.tsx [hunk 3]
```diff
@@ -49,14 +50,15 @@ export const LoadUnloadMapSettingsDisplay = ({
...
   ...otherProps
 }: MapSettingsDisplayProps) => {
+  const { t } = useTranslation()
   const { address, dateTime } = useLoadUnloadPageContext()
   const showAddressForm = () => setShowAddressForm(true)
   const showDateTimeModal = () => setShowDateTimeModal(true)
 
   return (
     <MapPanelContent data-testid="map-settings" {...otherProps}>
       <Tabs label="Voer adres en moment in of bekijk dataverantwoording">
```
Comment (TerryvanWalen) [inline src/pages/LoadUnload/components/MapSettingsDisplay.tsx:60]: We should also translate the label right?

File: src/pages/ProhibitorySigns/components/ScenarioDisplay/ScenarioDisplay.tsx [hunk 4]
```diff
@@ -53,7 +55,7 @@ const ScenarioDisplay = ({ ...otherProps }: MapPanelContentProps) => {
     <MapPanelContent data-testid="scenario-display" {...otherProps}>
       {!showScenarioWizard && (
         <Tabs label="Voer voertuiggegevens in of bekijk dataverantwoording">
```
Comment (TerryvanWalen) [inline src/pages/ProhibitorySigns/components/ScenarioDisplay/ScenarioDisplay.tsx:57]: We should also translate this label right?

File: src/pages/RoadObstructions/components/FiltersDisplay.tsx [hunk 5]
```diff
@@ -49,14 +50,16 @@ const RoadObstructionsFiltersDisplay = ({
...
 }: FiltersDisplayProps) => {
+  const { t } = useTranslation()
+
   function showMapFiltersForm() {
     setShowMapFiltersForm(true)
   }
 
   return (
     <MapPanelContent {...otherProps}>
       <Tabs label="Voer datum en tijd in of bekijk dataverantwoording">
```
Comment (TerryvanWalen) [inline src/pages/RoadObstructions/components/FiltersDisplay.tsx:61]: We should also translate this label right?

File: src/shared/components/Header/Header.test.tsx [hunk 6]
```diff
@@ -31,19 +31,4 @@ describe('Header', () => {
       screen.getByText('Title is optional', { selector: 'a' })
```
Comment (TerryvanWalen) [inline src/shared/components/Header/Header.test.tsx:31]: Did you set the testing language to dutch but then this should be in dutch too right?

**Review by TerryvanWalen — CHANGES_REQUESTED** [review]
Ziet er goed uit (sorrry comments in engels gedaan oops). Denk dat je mogelijk nog een paar plekken hebt waar text niet vertaald wordt. zoals in labels en tests, daar zou ik nog even naar kijken.

File: src/i18n/index.js (hunk 1 above)
Comment (RubenSibon) [reply src/i18n/index.js:32]: I made it true in `development` mode only.

**Comment by RubenSibon** [comment]
> Ziet er goed uit (sorrry comments in engels gedaan oops). Denk dat je mogelijk nog een paar plekken hebt waar text niet vertaald wordt. zoals in labels en tests, daar zou ik nog even naar kijken.

English comments are correct I think. For our open source projects the language should be English.

File: test/utils/withApp.tsx (hunk 2 above)
Comment (RubenSibon) [reply test/utils/withApp.tsx:15]: The tests already cover that. This file, `withApp.tsx`, is imported by those tests to use i18n and set the language we are testing on, Dutch in this case. If certain keys in the Dutch translations are missing or the file is completely missing the tests will fail.

File: src/pages/LoadUnload/components/MapSettingsDisplay.tsx (hunk 3 above)
Comment (RubenSibon) [reply src/pages/LoadUnload/components/MapSettingsDisplay.tsx:60]: Yeah, I'll add that.

File: src/shared/components/Header/Header.test.tsx (hunk 6 above)
Comment (RubenSibon) [reply src/shared/components/Header/Header.test.tsx:31]: No, this is just a dummy string to the test that a Header can receive an optional title.

**Review by TerryvanWalen — APPROVED** [review]
Looking good

**Pull request merged**